### ISSUE TITLE
Expected indentation of "NaN" spaces but found 0.

### ISSUE THREAD

**Issue by ScionOfBytes** [palantir/tslint#3077]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: 2.4.2
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
'use strict'

import { betalistUtil, logger as Logger } from '@crowdfire/cf-util'

import * as Constants from '../constants'

export const init = function () {
  return betalistUtil.initializeClient()
    .then(() => Logger.info(null, null, `Betalist initialized.`))
    .catch(err => {

      Logger.fatal(null, null, `Betalist failed to initialized.`, err)
      return false
    })
}

export const isUserQrEligible = function (userId: string) {
  return betalistUtil.isAllowed(userId, Constants.BETALIST.DAILY_IN_APP_QR)
    .catch(err => {

      Logger.err(null, userId, `Betalist encountered an error.`, err)
      return false
    })
}
```

with `tslint.json` configuration:

```json
{
  "extends": "tslint-config-airbnb",
  "rules": {
    "semicolon": [true, "never"],
    "trailing-comma": [false, {"multiline": "always", "singleline": "always"}],
    "max-line-length": [true, 120],
    "arrow-parens": [true, "ban-single-arg-parens"],
    "ter-arrow-parens": [true, "as-needed"],
    "object-shorthand-properties-first": false,
    "import-name": false
  }
}
```

#### Actual behavior

On the last line of the file I'm getting `Expected indentation of "NaN" spaces but found 0.`

#### Expected behavior

This linting error shouldn't happen :/

Note: This happened when I updated Typescript from 2.3.3 to 2.4.2. On 2.3.3 this doesn't happen.

**Comment by IllusionMH** [comment]
@ScionOfBytes looks like this is problem with [tslint-eslint-rules](https://github.com/buzinas/tslint-eslint-rules) package which is dependency of `tslint-config-airbnb` configuration.

Message template that you see can be found in [ter-indent on line 311](https://github.com/buzinas/tslint-eslint-rules/blob/master/src/rules/terIndentRule.ts#L311) but not in this repo.

Please report issue in [tslint-eslint-rules](https://github.com/buzinas/tslint-eslint-rules) repo.

**Comment by adidahiya** [comment]
Thanks for the input @IllusionMH, you're correct.
